### ISSUE TITLE
Automated Testing

### ISSUE THREAD

**Issue by mcav** [mozilla-sensorweb/sensorweb-client#10]
Presumably using Travis.

**Comment by isabelrios** [comment]
I was about to open an issue about this. In fact I started to think about this. I drafted a travis file at least to check that after new commits the build is successfully created. Not sure if it would work, will create a PR and check if that would be ok to start with having some automated checks. Then we would need to add lint errors check, unit and e2e tests as we create them.

**Comment by mcav** [comment]
Yes, that would be a good first step; it should be easy enough to confirm that the build works. We'd have to do a little more work to get the linter to validate (selecting the right rules for `tslint`).

**Comment by isabelrios** [comment]
Although travis file is included I do not see it being triggered by pushing commits that happened after it was added. Maybe it is necessary to do this, but with higher rights (I do not find that settings menu):
Go to your repository's settings page, click on "Webhooks & Services" on the left menu, choose "Travis CI" in the "Services", and use the "Test service" button.
@mcav Would you mind to check? Thanks!

**Comment by ferjm** [comment]
I just enabled this repo in Travis.

**Comment by isabelrios** [comment]
Thanks @ferjm!!
